libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -19,12 +19,23 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/**
+ * @file mqtt_lightweight.h
+ * @brief User-facing functions for serializing MQTT 3.1.1 packets. This header
+ * should be included for building a light-weight MQTT client bypassing the
+ * stateful CSDK MQTT library API in mqtt.h.
```

Comment:
Would it be accurate to call this stateful? I know we keep state records for QoS 1/2 publishes, but we don't keep track of acks for un/subscribes. We could just use "managed"
```suggestion
 * managed CSDK MQTT library API in mqtt.h.
```

Reply:
Sure